### DISCUSSION TITLE
I wish I could 'favorite' certain Traces

### DISCUSSION THREAD

**Discussion by StachuDotNet** [darklang/dark#4334] [Feedback]
Sometimes, I'll want to go back to a certain trace, but currently have to either:
- scroll down the 'dots' to the left of a handler until I find the trace in question, or
- give up hope, because the trace is no longer available in the UI

Being able to 'favorite' a trace, or 'pin' it somewhere, could help this

**Comment by pbiggar** [comment]
What's the difference between pinning and favoriting?

**Reply by StachuDotNet** [reply]
No difference - just different options for the term
